src/actions/channel_categories.ts

```diff
@@ -183,6 +179,10 @@ export function moveChannelToCategory(categoryId: string, channelId: string, new
 
         // And remove it from the old category
         const originalCategory = getCategoryInTeamWithChannel(getState(), category.team_id, channelId);
+        const sourceCategories = [{
+            ...category,
```

Comment:
Also, this can just be `category`. It doesn't need to be copied since we're not editing it